### ISSUE TITLE
Channel import from command line fails for latest 0.5.0 pex

### ISSUE THREAD

**Issue by ivanistheone** [learningequality/kolibri#2136]
## Summary

Running `python /kolibrihome/kolibri-v0.5.0.pex manage importchannel -- network 8a2d480dbc9b53408c688e8188326b16` fails on `self.check_for_cancel(last_stage)`

## System information
 - Version: 0.5.0 pex from github release
 - Operating system: linux

## Traceback or relevant snippet from server.log or browser console

```
[35.196.3.200] run: python /kolibrihome/kolibri-v0.5.0.pex manage importchannel -- network 8a2d480dbc9b53408c688e8188326b16

Fatal error: run() received nonzero return code 1 while executing!

Requested: python /kolibrihome/kolibri-v0.5.0.pex manage importchannel -- network 8a2d480dbc9b53408c688e8188326b16
Executed: /bin/bash -l -c "export KOLIBRI_HOME=\"/kolibrihome\" && python /kolibrihome/kolibri-v0.5.0.pex manage importchannel -- network 8a2d480dbc9b53408c688e8188326b16"

INFO     Version was 0.5.0, new version: 0.5.0
INFO     Running update routines for new version...

0 static files copied to '/kolibrihome/static', 379 unmodified.
Operations to perform:
  Apply all migrations: sessions, admin, kolibriauth, morango, auth, content, contenttypes, exams, logger
Running migrations:
  No migrations to apply.
  Your models have changes that are not yet reflected in a migration, and so won't be applied.
  Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
INFO     Downloading data for channel id 8a2d480dbc9b53408c688e8188326b16
  0%|                                                                                                                                                                                                               | 0/201728 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "/kolibrihome/kolibri-v0.5.0.pex/.bootstrap/_pex/pex.py", line 317, in execute
  File "/kolibrihome/kolibri-v0.5.0.pex/.bootstrap/_pex/pex.py", line 250, in _wrap_coverage
  File "/kolibrihome/kolibri-v0.5.0.pex/.bootstrap/_pex/pex.py", line 282, in _wrap_profiling
  File "/kolibrihome/kolibri-v0.5.0.pex/.bootstrap/_pex/pex.py", line 360, in _execute
  File "/kolibrihome/kolibri-v0.5.0.pex/.bootstrap/_pex/pex.py", line 418, in execute_entry
  File "/kolibrihome/kolibri-v0.5.0.pex/.bootstrap/_pex/pex.py", line 423, in execute_module
  File "/usr/lib/python2.7/runpy.py", line 180, in run_module
    fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/ivan/.pex/install/0.5.0/kolibri/__main__.py", line 8, in <module>
    main(args=sys.argv[1:])
  File "/home/ivan/.pex/install/0.5.0/kolibri/utils/cli.py", line 571, in main
    manage(command, args=django_args)
  File "/home/ivan/.pex/install/0.5.0/kolibri/utils/cli.py", line 402, in manage
    execute_from_command_line(argv=argv)
  File "/home/ivan/.pex/install/0.5.0/kolibri/dist/django/core/management/__init__.py", line 353, in execute_from_command_line
    utility.execute()
  File "/home/ivan/.pex/install/0.5.0/kolibri/dist/django/core/management/__init__.py", line 345, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/ivan/.pex/install/0.5.0/kolibri/dist/django/core/management/base.py", line 348, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/ivan/.pex/install/0.5.0/kolibri/dist/django/core/management/base.py", line 399, in execute
    output = self.handle(*args, **options)
  File "/home/ivan/.pex/install/0.5.0/kolibri/tasks/management/commands/base.py", line 98, in handle
    return self.handle_async(*args, **options)
  File "/home/ivan/.pex/install/0.5.0/kolibri/content/management/commands/importchannel.py", line 89, in handle_async
    self.download_channel(options["channel_id"])
  File "/home/ivan/.pex/install/0.5.0/kolibri/content/management/commands/importchannel.py", line 42, in download_channel
    self._transfer(DOWNLOAD_METHOD, channel_id)
  File "/home/ivan/.pex/install/0.5.0/kolibri/content/management/commands/importchannel.py", line 73, in _transfer
    if self.is_cancelled():
  File "/home/ivan/.pex/install/0.5.0/kolibri/tasks/management/commands/base.py", line 108, in is_cancelled
    self.check_for_cancel(last_stage)
TypeError: 'NoneType' object is not callable
```

Channel import works fine from the Web UI though.

**Comment by christianmemije** [comment]
Looks like #2082 which should have been fixed in #2093

**Comment by ivanistheone** [comment]
Ya, seems so. The 0.5 release I was using is older than that. Closing.
